src/sentry/notifications/notifications/activity/__init__.py

```diff
@@ -3,9 +3,6 @@
 from sentry.notifications.notifications.activity.assigned import AssignedActivityNotification
 from sentry.notifications.notifications.activity.base import ActivityNotification
 from sentry.notifications.notifications.activity.escalating import EscalatingActivityNotification
-from sentry.notifications.notifications.activity.new_processing_issues import (
-    NewProcessingIssuesActivityNotification,
```

Comment:
Can we actually delete this entire class as well then?

Looks like we're only deleting the imports and the template, but not the class nor the tuple enum? (maybe we can add a comment there that this is no longer used?)

Reply:
Will do, I believe I missed that and picked it up in the followup #73912, though I might just merge those two PRs together.